src/components/graph/Graph.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
Currently the project is set to have a file length limit of 450 lines. I tried pretty hard to keep this file under but couldn't do it without changing up a lot more things than needed.